src/AppBundle/Resources/translations/validators.en.yml

```diff
@@ -180,6 +176,27 @@ document:
       generic: "Cannot upload file, please try again later"
   wishToProvideDocumentation:
       notBlank: Please select either 'Yes' or 'No'
+clientContact:
+  form:
+    firstname:
+        notBlank: Enter the contact's first name
+        minMessage: The first name must be at least {{ limit }} letters long
+        maxMessage: The first name can't be longer than {{ limit }} letters
+    lastname:
+        notBlank: Enter the contact's last name
+        minMessage: The last name must be at least {{ limit }} letters long
+        maxMessage: The last name can't be longer than {{ limit }} letters
+    phone:
```

Comment:
Don't think this Phone validator is required for Contact

Reply:
yup, will never be used. removed